video2dataset/main.py

```diff
@@ -48,6 +48,9 @@ def video2dataset(
     tmp_dir: str = "/tmp",
     yt_metadata_args: dict = None,
     captions_are_subtitles: bool = False,
+    cut_detection_mode: str = None,
+    cut_framerates: list = None,
```

Comment:
make sure this works if None is passed i.e. cut detection happens on original fps (if you already checked ignore this comment)

Reply:
im assuming you're referring to cut_framerates? in which case yea ive tested it a lot.